src/providers/instagram/client.ts

```
@@ -0,0 +1,270 @@
+    );
+    return readSetCookieNames(res.headers).get('csrftoken') ?? null;
+  } catch {
+    return null;
+  }
+}
+
+/**
+ * One logged-out navigation to the homepage to collect cookies (`csrftoken`, `mid`, `ig_did`, …).
+ * Post permalink HTML often does not embed `xdt_api__v1__media__shortcode__web_info` without them.
+ */
+export async function fetchInstagramLoggedOutSession(userAgent: string | undefined): Promise<string> {
```

ESLint

[error] Replace `userAgent:·string·|·undefined` with `⏎··userAgent:·string·|·undefined⏎`

(prettier/prettier)